tests/test_odgovlt.py

```diff
@@ -0,0 +1,48 @@
+from ckanext.harvest.harvesters.ckanharvester import CKANHarvester
+from ckanext.harvest.harvesters.ckanharvester import DatetimeEncoder
+from sqlalchemy import MetaData
+from sqlalchemy.engine.base import Engine
+from datetime import datetime
+
+
+def test_info():
+    ckan_object = CKANHarvester()
+    info = ckan_object.info()
+    assert isinstance(info, dict)
+    assert isinstance(info['name'], str)
+    assert isinstance(info['title'], str)
+    assert isinstance(info['description'], str)
+    assert isinstance(info['form_config_interface'], str)
+
+
+def test_connect_to_database(mocker):
+    ckan_object = CKANHarvester()
+    config = mocker.patch('ckanext.harvest.harvesters.ckanharvester.config')
+    config.__getitem__.side_effect = 'mysql://test'.split()
+    mocker.patch('ckanext.harvest.harvesters.ckanharvester.MetaData')
+    con, meta = ckan_object._connect_to_database()
```

Comment:
I'm not sure why you test this protected method, this method will be called anyway, when you do test the other parts. So I would suggest to start with all the primary functionality and only add more tests for various helper methods if you need to increase test coverage.